build.ps1

```diff
@@ -155,24 +159,26 @@ $content = @(
     ,("$PSScriptRoot/src/Pester.psd1", "$PSScriptRoot/bin/")
 )
 
+Copy-Content -Content $content
+
+
 if ($Clean) {
-    $content += @(
+    $manifest = Test-ModuleManifest -Path "$PSScriptRoot/bin/Pester.psd1"
```

Comment:
This fails on PS3 with "The wildcard is incorrect"

Reply:
Well that was weird. Looks like `Test-ModuleManifest` invoked `Import-Dependency` for some absurd reason. 😕 
Stacktrace?

Reply:
I don't have any, I can only see it not fail when running on ps7.

Reply:
Reverted to PS3 build with (ugly) workaround in case PS3 build is still required. If it's okay to use PS7 for build then we should use `Test-ModuleManifest` again.

The PS7/vs2019 run only failed due to the psversion-output commit. It made passthru object `$r` from p-tests become arrays which broke the if failed-check after.

Reply:
We can build on ps7, but I would keep both approaches in case I need to debug something specifically on PS3. Thx. And thanks for helping here, I had no idea what was happening. 🙂